apps/src/templates/GameButtons.jsx

```diff
@@ -19,12 +21,30 @@ export const FinishButton = () => (
   </button>
 );
 
+const sendAnalyticsEvent = () => {
+  const isSignedOut =
+    document
+      .querySelector('script[data-issignedout="true"]')
+      ?.getAttribute('data-issignedout') === 'true';
+
+  if (isSignedOut) {
+    analyticsReporter.sendEvent(
+      EVENTS.RUN_BUTTON_PRESSED_SIGNED_OUT,
```

Comment:
This was on a Game Lab project:

<img width="384" alt="Screenshot 2024-07-08 at 1 31 13 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/c65687c1-266f-4f09-834f-d2e61536d08d">